runtime/tesla/org.eclipse.rcptt.tesla.swt/src/org/eclipse/rcptt/tesla/internal/ui/player/TeslaSWTAccess.java

```diff
@@ -1,19 +1,21 @@
 /*******************************************************************************
- * Copyright (c) 2009, 2019 Xored Software Inc and others.
+ * Copyright (c) 2009, 2019, 2022 Xored Software Inc and others.
```

Comment:
Better change to:
`Copyright (c) 2009, 2022 Xored Software Inc and others.`

Reply:
Done.